### ISSUE TITLE
Switching with left click, Oculus or Steam VR

### ISSUE THREAD

**Issue by sutex** [TuTAH1/SteamVR-Toggle-for-Viveport#6]
I'm not sure how this is intended to work.

The click icon switcher to Oculus or SteamVR seems to have to be set before launching your HMD?

With Oculus Dash in the HMD, and clicking the icon to steam VR, nothing happens. You have to start Steam Vr yourself first. Or shut down Oculus Home, then click the icon for Steam VR. Once Oculus Home is restarted, the HMD is in SteamVR mode, and Oculus Dash is no longer showing in the HMD.

Sometimes in this testing, I would get this.

![2023-04-23 16_28_21-Error](https://user-images.githubusercontent.com/10361359/233824894-4462a703-b1e3-4eb0-85c8-46417cea3ade.png)
![2023-04-23 16_11_43-](https://user-images.githubusercontent.com/10361359/233824898-91e5cd24-a4e0-4c0a-8d96-7867b8ef7edb.png)

Can you please translate,

**Comment by TuTAH1** [comment]
> The click icon switcher to Oculus or SteamVR it seems it has to be set before launching your HMD?

It can be done in any time, but you can see the result only after connecting to the PC (via Oculus Link or Oculus Airlink)

The "access path denied" is shown, becouse Windows don't let my program change files. Try to launch the program as Administrator. In my case, it solves all the issues

**Comment by TuTAH1** [comment]
Duplicate of #2
